README.md

```diff
@@ -23,6 +23,29 @@ rosinstall --catkin . uwreact_robot/uwreact_robot.rosinstall
 rosdep install --from-paths . --ignore-src -r -y
 ```
 
+In order to build anything zed related, you'll also need to install CUDA 10 and the ZED SDK
+
+**CUDA 10**
+
+Go to: https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&target_distro=Ubuntu&target_version=1804&target_type=deblocal and follow install instruction according to your device & OS
+Once installed, add these to your ~/.bashrc file
+```
+export PATH=$PATH:/usr/local/cuda-10.0/bin
+export CUDADIR=/usr/local/cuda-10.0
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda-10.0/lib64
+```
+
+**ZED SDK**
+
+Go to: https://www.stereolabs.com/developers/release/#sdkdownloads_anchor and dowload the appropriate file
```

Comment:
Same as above:

Must make this a markdown link, saying something like:

[Download the appropriate ZED SDK](https://www.stereolabs.com/developers/release/#sdkdownloads_anchor)

```
 [Download the appropriate ZED SDK](https://www.stereolabs.com/developers/release/#sdkdownloads_anchor)
```